src/graph.jl

```diff
@@ -39,6 +39,35 @@ function min_bounding_rect(coords::Vector{Vector{Vector{Vector{Float64}}}})::Tup
 end
 
 
+function create_rtree(minimum_bounding_rects::Array{Tuple{Array{Float64,1},Array{Float64,1}},1})::LibSpatialIndex.RTree
+    """ Given an array of minimum bounding rectangles, constructs an R-Tree
+        that will make it easier to identify candidates for intersecting nodes.
```

Comment:
Could you explain the `minimum_bounding_rects` argument in more detail?

Reply:
Yes! will add to this docstring 😄